Parsers/ASimRegistryEvent/test/VMwareCarbonBlackCloud_vimRegistryEvent_DataTest.csv

```diff
@@ -0,0 +1,8 @@
+Result
+"(0) Error: 1 invalid value(s) (up to 10 listed) in 4338747 records (100.0%) for field [EventProduct] of type [Enumerated]: [""Carbon Black Cloud""] (Schema:RegistryEvent)"
+"(0) Error: 2 invalid value(s) (up to 10 listed) in 2957602 records (68.17%) for field [EventType] of type [Enumerated]: [""RegistryValueSet"",""RegistryValueDeleted""] (Schema:RegistryEvent)"
```

Comment:
can you please correct this string in ASIM tester csv?
![image](https://github.com/Azure/Azure-Sentinel/assets/97222872/36610003-0487-4504-908c-eb203e4f96d2)

Reply:
Done